.github/workflows/rust.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         cargo +$WASM_BUILD_TOOLCHAIN --version
         rustc +$WASM_BUILD_TOOLCHAIN --version
       env:
-        WASM_BUILD_TOOLCHAIN: nightly-2021-07-06
+        WASM_BUILD_TOOLCHAIN: stable-2021-11-01 # 1.56.1
```

Comment:
Why not just `1.56.1`?

Reply:
I messaged you directly. I can't get cargo version working right. Please help me get this passing :pray:

Reply:
Add below this line: `RUSTC_BOOTSTRAP: 1` and same for integration and lint.